blueocean-web/package.json

```diff
@@ -42,15 +42,20 @@
     "window-handle": "1.0.0"
   },
   "jenkinscd": {
-    "extDependencies": [
+    "import": [
       "history",
       "immutable",
-      "react-router",
       "keymirror",
       "react-redux",
       "react-router",
       "redux",
       "redux-thunk"
+    ],
```

Comment:
that sounds like a worthy experiment, maybe note it for later and can do some A/B with just those changes in future?